index.html

```diff
@@ -310,6 +310,16 @@ <h3>Authorization</h3>
 or legacy authorization technologies.
       </p>
 
+      <section>
+        <h4>Open Authorization (OAuth)</h4>
+        <p>
+If Open Authorization is utilized for authorization, version 2.0 of the
+protocol (OAuth 2.0) [[RFC6749]] MUST be used. The access tokens utilized by
+clients MAY be OAuth 2.0 Bearer Tokens [[RFC6750]] or any other valid OAuth 2.0
+token type.
```

Comment:
```suggestion
        <h4>OAuth 2.0</h4>
        <p>
If the OAuth 2.0 Authorization Framework [[RFC6749]] is utilized for authorization,
the access tokens utilized by clients MAY be OAuth 2.0 Bearer Tokens [[RFC6750]]
or any other valid OAuth 2.0 token type. Any valid OAuth 2.0 grant type MAY be used
to request the access tokens.
```

Reply:
> It makes no sense to explicitly disallow OAuth 1.0 if someone's crazy enough to use it.

The reason not to use it is embedded in your response. "If someone's crazy enough to use it" -- we don't want crazy usages of the API, thus, just ban OAuth 1.0. There is no one that we know of in their right mind that would use OAuth 1.0, right?

> The name of the protocol is not "Open Authorization".

You should tell that to the Wikipedia page:

https://en.wikipedia.org/wiki/OAuth

... and lots of articles on the Web:

https://www.google.com/search?q=%22open+authorization%22

That said, I'm fine w/ the change to not spell it out, whatever it means.

I'm concerned that we don't have any normative statements from this language that will make their way into a test suite. @OR13, @mprorock, @jricher, thoughts on that?